internal/host/hostcommands/logs_cmd.go

```diff
@@ -85,6 +85,7 @@ func createUploadLogsCmd(host *models.Host, baseURL, agentImage, mastersIPs stri
 	}
 	cmdArgsTmpl += "timeout 1h podman run --rm --privileged --net=host " +
 		"-v /run/systemd/journal/socket:/run/systemd/journal/socket -v /var/log:/var/log " +
+		"{{if .BOOTSTRAP}} -v /root/.ssh:/root/.ssh -v /tmp:/tmp {{end}}" +
```

Comment:
Why do you need /tmp?

Reply:
because installer-gather.sh use host's /tmp to put the result of the collection. I write to the same place and then installer-gather runs and bundle all the logs together seamlessly.

Reply:
Alright, though I don't think you need to collect ovs logs on the bootstrap.

Reply:
The ovs logs are not collected from bootstrap but from bootstrap. The script runs on the bootstrap and need access to the /tmp file system and then SSH to